src/Symfony/Bundle/WebProfilerBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+3.3.0
+----
+
+ * added ability to auto-refresh the toolbar if a "x-debug-toolbar-replace" header is set
```

Comment:
Prefixing custom headers with a `X-` is deprecated: https://tools.ietf.org/html/rfc6648
May I suggest `Symfony-Debug-Toolbar-Replace`?

Reply:
Are there any other examples in Symfony of not using an `X-` prefixed header? Or would this be the first place to adopt this new RFC recommendation?

Reply:
AFAIK it will be the first place.

Reply:
This being the case, do we need "higher level" input? The RFC you mention has been around since 2012 and doesn't seem to have been adopted elsewhere in SF. This change seems like something that should be done framework-wide, not just here in the WDT.

Reply:
IMO it should be done for all new features (for existing feature, it's a BC break and it's not so easy). Symfony is committed to respect web standards, including this one. We shouldn't introduce new code following a deprecated practice.

Reply:
Changed and ready for review.